Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -164,7 +167,7 @@ open class MapView: UIView {
     }
 
     deinit {
-        NotificationCenter.default.removeObserver(self)
+        notificationCenter.removeObserver(self)
     }
```

Comment:
We should just delete this. NotificationCenter keeps weak refs to observers on every OS version we support.

Reply:
Deleted.